Tests/Command/CreateDatabaseDoctrineTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: mike
+ * Date: 30/07/14
+ * Time: 16:48
+ */
+
+namespace Doctrine\Bundle\DoctrineBundle\Tests\Command;
+
+use Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand;
+use Doctrine\Common\Cache\ArrayCache;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CreateDatabaseDoctrineTest extends \PHPUnit_Framework_TestCase
+{
+    public function testExecute()
+    {
+        $connectionName = 'default';
+        $dbName = 'test';
+        $params = array(
+            'dbname' => $dbName,
+            'memory' => true,
+            'driver' => 'pdo_sqlite'
+        );
+
+        $application = new Application();
+        $application->add(new CreateDatabaseDoctrineCommand());
+
+        $command = $application->find('doctrine:database:create');
+        $command->setContainer($this->getMockContainer($connectionName, $params));
+
+        $commandTester = new CommandTester($command);
+        $commandTester->execute(
+            array_merge(array('command' => $command->getName()))
+        );
+
+        $this->assertRegExp("/Created database \"$dbName\" for connection named $connectionName/", $commandTester->getDisplay());
```

Comment:
you could use ``assertContains`` instead of usign a regex. It will be simpler (and the failure message will be more understandable in case of failure)